src/routes/docs/configuring/custom-themes/+page.md

```diff
@@ -32,7 +32,7 @@ Files has built-in support for custom fonts, to apply a custom font start by ope
 
 We put together some popular color combinations to help users create custom themes.
 
-#### **Glass (requires Windows 11)**
+#### **Glass**
```

Comment:
```suggestion
#### **Glass** (requires enabling the Acrylic backdrop on Windows 10)
```

Reply:
Maybe this note should be farther down as not to clutter up the title?

Reply:
@heftymouse

Reply:
That makes sense, it can be below the table